akka-stream-typed/src/test/scala/akka/stream/typed/scaladsl/CustomGuardianAndMaterializerSpec.scala

```diff
@@ -6,29 +6,14 @@ package akka.stream.typed.scaladsl
 
 import akka.Done
 import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
-import akka.actor.typed.ActorRef
 import akka.actor.typed.scaladsl.Behaviors
 import akka.stream.AbruptStageTerminationException
 import akka.stream.scaladsl.{ Sink, Source }
 import org.scalatest.WordSpecLike
 
 import scala.concurrent.Future
 
-object CustomGuardianAndMaterializerSpec {
-
-  sealed trait GuardianProtocol
-  case class Init(sender: ActorRef[String]) extends GuardianProtocol
-  case class Msg(sender: ActorRef[String], msg: String) extends GuardianProtocol
-  case object Complete extends GuardianProtocol
-  case object Failed extends GuardianProtocol
-}
-
 class CustomGuardianAndMaterializerSpec extends ScalaTestWithActorTestKit with WordSpecLike {
-  import CustomGuardianAndMaterializerSpec._
-
-  val guardian = Behaviors.receive[GuardianProtocol] { (_, msg) =>
-    Behaviors.same
-  }
```

Comment:
I wonder what the original purpose of this test was? Perhaps rewrite `"work with typed ActorSystem with custom guardian"` to actually start another ActorSystem with this guardian?

Reply:
Yes i was curious, and the actor system does have a custom guardian as it is the test kit's one